src/libaktualizr/uptane/imagerepository.cc

```diff
@@ -278,9 +294,14 @@ void ImageRepository::updateMeta(INvStorage& storage, const IMetadataFetcher& fe
     std::string image_targets_stored;
     if (storage.loadNonRoot(&image_targets_stored, RepositoryType::Image(), Role::Targets())) {
       try {
-        verifyTargets(image_targets_stored, true);
+        verifyTargets(image_targets_stored, true, snapshot_updated);
         fetch_targets = false;
         LOG_DEBUG << "Skipping Image repo Targets download; stored version is still current.";
+      } catch (const Uptane::SecurityException& e) {
+        if (snapshot_updated)
+          LOG_INFO << "Image repo Target verification failed: " << e.what();
```

Comment:
I believe `LOG_DEBUG` is fine. For a developer, the current message still makes sense, because it explicitates the step that leads to the fetching of the new metadata